ReleaseNotes.md

```diff
@@ -11,6 +11,8 @@ Post-training Quantization:
   - (OpenVINO) Changed default OpenVINO opset from 9 to 13.
 - Features:
   - (OpenVINO) Added 4-bit data-aware weights compression. For that `dataset` optional parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).
+  - (PyTorch) Added `nncf.torch.wrap_model()` method to add tracing and modification capabilities to the PyTorch model.
+  - (PyTorch) Added support for PyTorch models with shared weights and custom PyTorch modules in `nncf.compress_weights()`. The `dataset` parameter is now a required input for PyTorch models when using `nncf.compress_weights()`. This is a result of migrating the compression weights algorithm for PyTorch models to an approach based on tracing the model graph, where the `dataset` parameter is utilized to obtain an example input.
```

Comment:
(PyTorch) Added support for PyTorch models with shared weights and custom PyTorch modules in `nncf.compress_weights()`. The weights compression algorithm for PyTorch models is now based on tracing the model graph. The `dataset` parameter is now required in `nncf.compress_weights()` for the compression of PyTorch models.

Reply:
done